src/rename/rename_function.py

```diff
@@ -30,6 +37,9 @@ def lambda_rename_dbinstance(event, context):
 
```

Comment:
log the response of the `modify_db_instance` call

Reply:
@namitad it doesn't look like the response value for modify_db_instance is currently being captured in the code. Are you wanting me to store the response then return it? Or am I just missing the line of code that specifies the response value?

Reply:
you can store the response and log it https://github.com/intuit/Trapheus/blob/c696939b800c28136e0a96dec44b95469d50a9dd/src/rename/rename_function.py#L33
no need to return it